src/taipy/rest/api/resources/cycle.py

```diff
@@ -34,7 +34,10 @@ class CycleResource(Resource):
       tags:
         - api
       summary: Get a cycle
-      description: Get a single cycle by ID
+      description: |
+        Return a single cycle by CycleId. If the cycle does not exist, a 404 error is returned.
```

Comment:
"Return a single cycle by CycleId"
->
"Returns the cycle identified by *cycle_id*."